javascript-conditionals/main.js

```diff
@@ -0,0 +1,97 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function startsWithJ(string) {
+  if (string.charAt(0) === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age <= 21) {
+    return false;
+  } else {
+    return true;
+  }
+}
+
+function isOldEnoughToDrive(person) {
+  if (person.age > 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'The pH is neutral';
+  } else if (pH >= 0 && pH <= 6) {
```

Comment:
What if the pH is 6.5 (or any other number between 6 and 7)?

Reply:
Looks like I need to brush up on my chemistry as well. This was under the assumption pH levels are only whole numbers!